File: Strategy/python-source/behaviours/DuckHuntFlyBehaviour.py
from __future__ import annotations

from .FlyBehaviour import FlyBehaviour


class DuckHuntFlyBehaviour(FlyBehaviour):
    """Утка из игры летает иначе, и утка про такое отличие не знает."""

    def execute(self) -> None:
        print("Duck is flying away from the hunter")

File: JavaSource/src/main/java/ru/akhcheck/patterns/strategy/behaviours/DuckHuntFlyBehaviour.java
package ru.akhcheck.patterns.strategy.behaviours;

/** Утка из игры летает иначе, и сама утка про это отличие не знает. */
public class DuckHuntFlyBehaviour implements FlyBehaviour {
    @Override
    public void execute() {
        System.out.println("Duck is flying away from the hunter");
    }
}
